src/__tests__/accessiblity/accessibility.test.tsx

```diff
@@ -14,11 +14,11 @@ import { renderWithRouter, store } from '../utils/setupTests';
 expect.extend(toHaveNoViolations);
 
 describe('Accessibility', () => {
-  beforeEach(() => {
+  beforeAll(() => {
```

Comment:
I don't get this, @oburasamuel. Could you explain why this change?
As a result of using `beforeAll()`, running the accessibility test (`test:a11y`) now times out.